packages/fetchai/connections/p2p_client/connection.py

```diff
@@ -63,7 +63,8 @@ def __init__(
         self.thread = Thread(target=self.receiving_loop)
         self.lock = threading.Lock()
         self.stopped = True
-        logger.info("Initialised the peer to peer channel")
+        self.logger = logger
```

Comment:
Why don't we delegate this to base?

Reply:
because the `logger` on the right is the module logger, which is accessible only in the inheriting class.